data/cube/Query.js

```diff
@@ -73,18 +81,32 @@ export class Query {
         return new Query(conf);
     }
 
+    /** @return {string} */
+    static filterAsString(filter) {
+        const {field, operator, value} = filter;
+        return `${field}${operator}[${castArray(value).join('||')}]`;
+    }
```

Comment:
Does this belong here or is this something more general ?  Why not instance method on FieldFilter?